itou/www/siae_evaluations_views/tests/test_views.py

```diff
@@ -109,6 +117,13 @@ def test_view_as_siae(self):
             )
         )
         self.assertSanctionContent(response)
+        self.assertNotContains(
+            response,
+            '<a class="btn btn-primary float-right" '
+            f'href="/siae_evaluation/institution_evaluated_siae_list/{self.evaluated_siae.evaluation_campaign_id}/">'
+            "Retour</a>",
```

Comment:
Idéalement, il faudrait appairer cette chaine avec l’assertion positive. Autrement, le moindre changement à la chaine qui ne serait pas reporté dans l’assertion négative la rendra caduque.

```suggestion
            response,
            self.return_link_html,
```

Reply:
Corrigé, je pense.